samples/scaleout/distributor/sample.md

```diff
@@ -73,6 +73,7 @@ snippet: workerConfig
 
 The Node in the MasterNodeConfig points to the host name where the MasterNode is running. If running the Worker from the same machine as the Distributor, Node should equal "localhost".
 
+NOTE: As of Version 6 `MasterNodeConfig` is obsolete and `endpointConfiguraiton.EnlistWithLegacyMSMQDistributor` should be used as in code above. 
```

Comment:
@WojcikMike - I am wondering if we should make this sample obsolete with a similar warning to what we did here:
http://docs.particular.net/samples/sql-bridge/, i.e. `This sample has been deprecated. For Versions 6 and above, see []`
and point to a newer sample that shows how to do dynamic scaling using the new v6 features. 

Thoughts? cc / @SzymonPobiega